### PR TITLE
Add example and lint

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -59,7 +56,53 @@ To install in a virtual environment in your current project:
...
+    import busio
+    import displayio
+    import adafruit_ssd1608
+
+    displayio.release_displays()
+
+    # This pinout works on a Metro and may need to be altered for other boards.
+
+    # For breadboarding
+    spi = busio.SPI(board.SCL, board.SDA)
```
Comment (makermelissa) [inline README.rst]: Let's change this to board.SPI() instead of using the I2C pins.

**Review by makermelissa — CHANGES_REQUESTED** [review]
Tested good. The only thing is the use of the I2C pins for SPI.

**Comment by tannewt** [comment]
Ok, swapped to the Feather breadboard setup that uses board.SPI.

**Review by makermelissa — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**